packages/host/app/components/editor/file-tree.gts

```diff
@@ -71,19 +105,31 @@ export default class FileTree extends Component<Signature> {
       .realm-info img {
         width: 18px;
       }
+
+      .editability-icon {
+        display: flex;
+      }
     </style>
   </template>
 
   @service private declare router: RouterService;
   @tracked private showMask = true;
+
+  private realmResource: RealmResource | undefined;
+
   constructor(owner: Owner, args: Signature['Args']) {
     super(owner, args);
     this.hideMask.perform();
+    this.realmResource = getRealm(this, () => this.args.realmURL);
```

Comment:
just FYI, the API for the realm resource changed a little bit in a recent PR. after you merge upstream, this now looks like:
```ts
    this.realmResource = getRealm(this, { realmURL: () => this.args.realmURL });
```